sqlmesh/dbt/adapter.py

```diff
@@ -311,7 +311,10 @@ def resolve_identifier(self, relation: BaseRelation) -> t.Optional[str]:
     def _map_table_name(
         self, database: t.Optional[str], schema: t.Optional[str], identifier: t.Optional[str]
     ) -> exp.Table:
-        name = ".".join(p for p in (database, schema, identifier) if p is not None)
+        name = normalize_model_name(
+            exp.table_(identifier or "", db=schema, catalog=database),
```

Comment:
is this gonna work? "" as a default will just crash won't it? or is this just to trick mypy?